.nvmrc

```diff
@@ -1 +1 @@
->=12.22
+lts/*
```

Comment:
We should pin to some version and periodically update, no? As it stands here, a contributor could come into the repo and install Node 20 someday and it could have compatibility issues.

Reply:
I don't know how to allow multiple versions easily, or maybe we just remove it to make `engines` do the job?

Reply:
Well `.nvmrc` is just meant to define the node version for contributors! I think it's okay to use the best, latest version that we support.

Reply:
Then what value you'd like it to be here? `>=12.22` is broken because nvm thinks it is a single version. 🤣

Reply:
Yeah >=12.22 is the correct value for the `engines` field in `package.json`

For us, I guess just the most recent version of v18 since we support 12-18. An argument could be made for latest minor of v16 because it's LTS and 18 isn't. Take your pick!